tech_soln/real_time_analytics.rst

```diff
@@ -0,0 +1,425 @@
+.. _introduction:
+
+Real Time Analytics
+#####################
+
+One workload Citus is commonly used for involves running real-time analytics over large
+streams of data.
+
+For example, say you're a cloud services provider selling a dashboard for HTTP traffic.
+Every time one of your clients receives an HTTP request your service receives a log
+record. You provide your clients insights such as the number of HTTP errors their site
+served. It's important that this data shows up with as little latency as possible, so your
+clients can quickly notice problems with their sites. It's also important that clients are
+able to see graphs of historical trends.
+
+Alternatively, maybe you're building an advertising network and want to show clients
+clickthrough rates on their campaigns. In this example latency is also critical, raw data
+volume is also high, and both historical and live data are important.
+
+In this technical solution we'll demonstrate how to build part of the first example,
+but this architecture would work equally well for the second and many other use-cases.
+
+Running It Yourself
+-------------------
+
+There will be code snippets in this tutorial but they don't specify a complete system.
+There's `a github repo <https://github.com/citusdata/technical-solution-resources>`_ with
+all the details in one place. If you've followed our installation instructions for running
+Citus on either a single or multiple machines you're ready to try it out.
+
+Data Model
+----------
+
+The data we're dealing with is an immutable stream of log data. We'll insert directly into
```

Comment:
I suppose, but why do you want to phrase it like this? Kafka is useful for way more than just enabling batching, and the way this is phrased makes it sound like Kafka itself does the batching.

I had originally thought you just wanted me to trim down how much time I spent talking about why Kafka might be useful. I could just get rid of the reference to Kafka entirely.